### PR TITLE
add ability to submit for w/o js

### PR DESCRIPTION
**Pull request by dreamfall**

### CODE REVIEW

File: app/controllers/form_controller.rb [hunk 1]
```diff
@@ -91,31 +89,48 @@ def edit_form
     end
   end
 
-  def submit_form
-    doc = params[:form]
-    @form_answer.document = serialize_doc(doc)
+  def save
```
Comment (matthewford) [inline app/controllers/form_controller.rb]: Assignment Branch Condition size for save is too high. [33.67/15]

File: app/controllers/form_controller.rb [hunk 2]
```diff
@@ -91,31 +89,48 @@ def edit_form
...
+          if submitted
+            redirect_to submit_confirm_url(@form_answer)
+          else
+            redirect_to edit_form_url(@form_answer.id, step: params[:next_step])
+          end
+        end
+      end
+
+      format.js do
+        render json: {progress: @form_answer.fill_progress}
```
Comment (matthewford) [inline app/controllers/form_controller.rb]: Space inside { missing.<br>Space inside } missing.

File: config/routes.rb [hunk 3]
```diff
@@ -25,8 +25,7 @@
   get  "/new_sustainable_development_form"              => "form#new_sustainable_development_form",                       as: "new_sustainable_development_form"
   get  "/new_enterprise_promotion_form"                 => "form#new_enterprise_promotion_form",                          as: "new_enterprise_promotion_form"
   get  "/form/:id"                                      => "form#edit_form",                                              as: "edit_form"
-  post "/form_autosave/:id"                             => "form#autosave",                                               as: "autosave"
-  post "/form/:id"                                      => "form#submit_form",                                            as: "submit_form"
+  post "/form/:id"                                      => "form#save",                                                   as: "save_form"
```
Comment (matthewford) [inline config/routes.rb]: Line is too long. [137/100]<br>Unnecessary spacing detected.

**Comment by RuslanHamidullin** [comment]
+1, looks like need to rebase @dreamfall

**Pull request merged**